frc5024/InfiniteRecharge
# Add DriveTrain code, position tracking, and autonomous position setting

We added the following:
 - DriveTrain control code
 - Position tracking (robot now knows where it is on the field)
 - Autonomous selectors
 - Position setting

## Testing
We should test this code out on MiniBot this week and ensure the following:
 
 1) Make sure the robot can be driven
 2) Make sure the position data is correct
     - This can be tested by printing `DriveTrain.getPosition()` to the log, and making sure the data looks reasonable
